Assets/Scripts/StandardSamples/Discord/DiscordManager.cs

```diff
@@ -157,8 +200,11 @@ private void GetTokenAsyncComplete(Task<TokenReader> tokenTask)
             if (tokenContent != null)
             {
                 cachedToken = tokenContent.access_token;
-                //in a real application the refresh token would be stored as well and used to refresh the auth token
+
                 tokenExpiration = DateTime.Now + TimeSpan.FromSeconds(tokenContent.expires_in-60);
+                
+                // Set the refresh token so next grant attempt can try to use it
+                refresh_token = tokenContent.refresh_token;
```

Comment:
What happens if multiple calls are made and the `refresh_token` value is overwritten? Is that a scenario that could cause problems, or is the nature of it such that it wouldn't be a problem?

Reply:
Noting that this is likely a problem, but we consider it outside the scope of this PR. I'll note this for later in ClickUp.